dev-support/hbase-personality.sh

```diff
@@ -146,7 +146,7 @@ function personality_modules
   fi
 
   if [[ -n "${HADOOP_PROFILE}" ]]; then
-    extra="${extra} -Dhadoop.profile=${HADOOP_PROFILE}"
+    extra="${extra} -Phadoop-${HADOOP_PROFILE}"
```

Comment:
Why this change? Makes it easier than what was there previous when you want to swap profiles?

Reply:
For maven it is hard to add multiple activating conditions so on master branch, the hadoop-3.0 profile can only be actived without specifying hadoop.profile. In general, using -Phadoop-3.0 should work on all branches but seems we have an enforcer rule for checking the property. Let me take a look.